pyrolite/data/geochem/refcomp/NASC_Gromet1984.csv

```diff
@@ -0,0 +1,95 @@
+var,value,units,unc_2sigma,constraint_var,constraint_value,constraint_comment
+ModelName,NASC_Gromet1984,,,,,
+Reservoir,NorthAmericanShaleComposite,,,,,
+ModelType,Average,,,,,
+Reference,"Gromet et al., 1984; Taylor and McLennan, 1985; Condie, 1993; McLennan, 2001",,,,,
```

Comment:
In the references for this file there's string which contains the file delimiter (a comma) - perhaps change this to remove commas and separate references with semicolons.